packages/fxa-payments-server/src/routes/Product/PayPalButton/index.tsx

```diff
@@ -1,19 +1,89 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import ReactDOM from 'react-dom';
 
+import * as apiClient from '../../../lib/apiClient';
+import { Customer } from '../../../store/types';
+import { SubscriptionCreateAuthServerAPIs } from '../SubscriptionCreate';
+
 declare var paypal: {
   Buttons: {
     driver: Function;
   };
 };
 
-const PaypalButtonBase = paypal.Buttons.driver('react', {
-  React,
-  ReactDOM,
-});
+export type PaypalButtonProps = {
+  apiClientOverrides?: Partial<SubscriptionCreateAuthServerAPIs>;
+  customer: Customer | null;
+  setPaymentError: Function;
+  idempotencyKey: string;
+  ButtonBase?: React.ElementType;
+};
+
+export type ButtonBaseProps = {
+  createOrder?: (
+    event: React.MouseEvent<HTMLButtonElement, globalThis.MouseEvent>
+  ) => void;
+  onError?: (
+    event: React.MouseEvent<HTMLButtonElement, globalThis.MouseEvent>
```

Comment:
I don't think we need `globalThis.` for the MouseEvent type.

Reply:
I think this autocompleted when we were pairing together -- why don't we need it?

Reply:
It's okay to leave it.  Or remove `globalThis.MouseEvent` altogether even, because apparently `React.MouseEvent` have a default type for that...which is just an alias to `globalThis.MouseEvent` (dun dun dun).

Reply:
Okay I removed it. Yeah I noticed that, heh.